docs/docs/networks/goerli.mdx

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 2
 id: goerli
 ---
-
+# UNSUPPORTED
```

Comment:
I think is better just to remove the file, since might be confusing for others if they don't read the `UNSOPPORTED`.